wallet/how-to/use-non-evm-networks/starknet/manage-starknet-accounts.md

```diff
@@ -0,0 +1,238 @@
+      ) : (
+        <p>No account connected</p>
+      )}
+      {available.map((connector) => (
+        <button key={connector.id()} onClick={() => connect(connector)}>
+          Connect {connector.name()}
+        </button>
+      ))}
+      {account && (
+        <button onClick={() => disconnect()}>Disconnect</button>
+      )}
+    </div>
+  );
+}
+```
+
+This component will display the connected account address if available, and provide buttons to connect or disconnect accounts.
+
+## 2. Switching Between Multiple Starknet Accounts
+
+Starknet doesn't natively support account switching like Ethereum. Instead, users typically have to disconnect and reconnect with a different account. However, you can implement a user-friendly interface to manage this process:
+
+```typescript
+import { useStarknet, useConnectors } from '@starknet-react/core';
+import { useState, useEffect } from 'react';
+
+function AccountSwitcher() {
+  const { account } = useStarknet();
+  const { available, connect, disconnect } = useConnectors();
+  const [connectedAccounts, setConnectedAccounts] = useState<string[]>([]);
+
+  useEffect(() => {
+    if (account?.address && !connectedAccounts.includes(account.address)) {
+      setConnectedAccounts([...connectedAccounts, account.address]);
+    }
+  }, [account]);
+
+  const switchAccount = async (connector: any) => {
+    await disconnect();
+    await connect(connector);
+  };
+
+  return (
+    <div>
+      <h2>Connected Accounts:</h2>
+      <ul>
+        {connectedAccounts.map((address) => (
+          <li key={address}>{address}</li>
+        ))}
+      </ul>
+      <h2>Available Connectors:</h2>
+      {available.map((connector) => (
+        <button key={connector.id()} onClick={() => switchAccount(connector)}>
+          Switch to {connector.name()}
+        </button>
+      ))}
+    </div>
+  );
+}
+```
+
+This component keeps track of all connected accounts and allows switching between available connectors.
+
+## 3. Account Creation Process
+
+Account creation in Starknet is typically handled by the wallet provider (e.g., ArgentX, Braavos). As a dApp developer, you don't create accounts directly. Instead, you guide users to create an account with their preferred wallet provider. Here's how you might implement a guide for users:
+
+```typescript
+function AccountCreationGuide() {
+  return (
+    <div>
+      <h2>Create a Starknet Account</h2>
+      <ol>
+        <li>Install a Starknet-compatible wallet (e.g., ArgentX, Braavos)</li>
+        <li>Open the wallet and follow its account creation process</li>
+        <li>Once created, return to this dApp and connect your new account</li>
+      </ol>
+      <p>After creating your account, you can connect it to this dApp.</p>
+    </div>
+  );
+}
+```
+
+## Handling Account Changes and Disconnections
+
+To handle account changes and disconnections, you can use event listeners provided by get-start. Here's an example using `@starknet-react/core`:
```

Comment:
The handling of disconnection is already present at the beginning of the tutorial in the handleDisconnect method. Maybe we could simply describe it here ?